src/components/Form.tsx

```diff
@@ -194,10 +230,28 @@ const Form: React.FC<FormProps> = ({setCardInfo}) => {
                         inputRef={capacityRef}
                         onChange={handleCapacityChange}
                     />
-                    <div className="error-message">{capacityError}</div>
-                </div>
-                <div className="input-container select-container">
-                    <label htmlFor="field-type">Choose field type:</label>
+                    {capacityError && (
+                        <Stack
+                            direction="row"
+                            gap={1}
+                            sx={{
+                                color: theme.palette.error.light,
+                            }}
+                        >
+                            <ErrorIcon fontSize="small" />
+                            <Typography variant="body2">
+                                {capacityError}
+                            </Typography>
+                        </Stack>
+                    )}
+                </Box>
+                <Box
+                    sx={{md: {minHeight: "105px"}}}
+                    className="input-container select-container"
+                >
+                    <InputLabel htmlFor="field-type">
```

Comment:
What do you think about using label prop from textfield. It will look like this 
![image](https://github.com/proga-react-practice/sigma-ivan-ostapiv/assets/74414048/f1245e74-9e7f-4c5a-8ac7-14bb51101ec5)
![image](https://github.com/proga-react-practice/sigma-ivan-ostapiv/assets/74414048/a4c09ecb-bffa-4235-94f8-68a255ae9127)
Then you can avoid using InputLabel and just pass label to your input component